scripts/validation/validate-version.js

```diff
@@ -0,0 +1,22 @@
+const { readFile } = require("fs").promises;
+const { join } = require("path");
+const { parseStringPromise } = require("xml2js");
+
+main().catch(error => {
+    console.error(error);
+    process.exit(1);
+});
+
+async function main() {
+    const packageXmlAsJson = await parseStringPromise(
+        (await readFile(join(process.cwd(), "src/package.xml"))).toString(),
+        { ignoreAttrs: false }
+    );
+    const packageXmlVersion = packageXmlAsJson.package.clientModule[0].$.version;
+    const packageJson = JSON.parse((await readFile(join(process.cwd(), "package.json"))).toString());
+    const packageJsonVersion = packageJson.version;
+
+    if (packageJsonVersion && packageXmlVersion && packageJsonVersion === packageXmlVersion) return;
```

Comment:
🎄 (nice to have)This if statement is also not entered when you miss the version field somewhere. It would be nice to output a specific error for that. When leaving it as is, the error below can be confusing.

Reply:
Agreed, this is useful. Changed